libs/auth/src/lib/login/login.component.html

```diff
@@ -1,77 +1,12 @@
 <!-- implements F011 -->
-<section *ngIf="authState$ | async as authState">
-  <ng-template #authdone>
-    <div *ngIf="authState.loading">
-      <div class="grid h-screen place-items-center">
-        <div role="status">
-          <party-time-loading-circle></party-time-loading-circle>
-        </div>
-        <p
-          class="m-10 h-full text-3xl font-extrabold text-on-background-light dark:text-on-background-dark md:text-5xl lg:pt-5 lg:text-6xl"
-        >
-          Login wird durchgeführt
-        </p>
-      </div>
-    </div>
-    <ng-template #error>
-      <div class="grid h-screen place-items-center">
-        <div
-          class="text-xl font-extrabold text-on-background-light dark:text-on-background-dark md:text-2xl lg:text-3xl"
-        >
-          <p class="">Hey!</p>
-          <p>Wir konnten dich leider nicht einloggen.</p>
-          <p>Status: {{ authState.error?.status }}</p>
-          <div>
-            Fehler:
-            <p class="text-error-light dark:text-error-dark">
-              {{ authState.error?.message }}
-            </p>
-            <li
-              class="text-error-light dark:text-error-dark"
-              *ngFor="let error of authState.error?.additionalInformation"
-            >
-              {{ error['name'] }} {{ error['message'] }}
-            </li>
-          </div>
-        </div>
-      </div>
-    </ng-template>
-    <div *ngIf="authState.isAuthenticated; else error">
-      <div class="grid h-screen place-items-center">
-        <div
-          class="text-xl font-extrabold text-on-background-light dark:text-on-background-dark md:text-2xl lg:text-3xl"
-        >
-          <p
-            class="dark:from-on-secondary-dark-dark mb-1 bg-gradient-to-r from-primary-light to-tertiary-light bg-clip-text text-transparent dark:to-on-tertiary-dark"
-          >
-            Hey!
-          </p>
-          <p>Du wurdest erfolgreich eingeloggt.</p>
-        </div>
-      </div>
-    </div>
-  </ng-template>
-  <div
-    *ngIf="
-      !authState.loading && !authState.isAuthenticated && !authState.error;
-      else authdone
-    "
-  >
-    <h1
-      class="m-10 h-full text-3xl font-extrabold text-on-background-light dark:text-on-background-dark md:text-5xl lg:pt-5 lg:text-6xl"
-    >
-      Login
-    </h1>
-    <form class="" (ngSubmit)="(onSubmit)" [formGroup]="loginForm">
+<ng-container *ngIf="vm$ | async as vm">
+  <div *ngIf="!vm.isAuthenticated; else authenticatied">
```

Comment:
```suggestion
  <div *ngIf="!vm.isAuthenticated; else authenticated">
```

@JulianTurner Seiteneffekte dieser Änderung?